src/components/modes/teacher/AssignBadgeForm.js

```diff
@@ -82,6 +83,7 @@ class AssignBadgeForm extends Component {
           options={badgeOptions}
         />
         <Button
+          className="AssignMedal"
```

Comment:
I would use `AssignBadge` as the class name, so that it is aligned with the rest of the nomenclature.